superset-frontend/spec/javascripts/components/AlteredSliceTag_spec.jsx

```diff
@@ -112,6 +113,28 @@ const expectedDiffs = {
   },
 };
 
+const expectedRows = [
+  {
+    control: 'Fake Filters',
+    before: 'a == hello',
+    after: 'b in [hello, my, name]',
+  },
+  {
+    control: 'Value bounds',
+    before: 'Min: 10, Max: 20',
+    after: 'Min: 15, Max: 16',
+  },
+  {
+    control: 'Fake Collection Control',
+    before: '{"1":"a","b":["6","g"]}',
+    after: '{"1":"a","b":[9,"15"],"t":"gggg"}',
+  },
+  { control: 'bool', before: 'false', after: 'true' },
+  { control: 'gucci', before: '1, 2, 3, 4', after: 'a, b, c, d' },
+  { control: 'never', before: 5, after: 10 },
+  { control: 'ever', before: '{"a":"b","c":"d"}', after: '{"x":"y","z":"z"}' },
+];
+
```

Comment:
not sure what our overall philosophy is on snapshots, but moving this to a snapshot could clean up this file a bit. Just a thought.

Reply:
I moved all mocked values from AlteredSliceTag_spec.jsx to separate file